docs/secure-comms.asciidoc

```diff
@@ -0,0 +1,25 @@
+[[securing-apm-server]]
+== Secure communication with the {stack}
+
+++++
+<titleabbrev>Secure communication</titleabbrev>
+++++
+
+The following topics provide information about securing the APM Server
+process and connecting securely to APM agents and the {stack}.
+
+* <<secure-agent-communication,Secure communication with APM agents>>
+* <<secure-comms-stack,Secure communication with the {stack}>>
+
+:leveloffset: +1
+include::secure-agent-communication.asciidoc[]
+
+// APM privileges
+include::{docdir}/legacy/feature-roles.asciidoc[]
+
+// APM API keys
+include::{docdir}/legacy/api-keys.asciidoc[]
+
+// Linux Seccomp
+include::{docdir}/legacy/copied-from-beats/docs/security/linux-seccomp.asciidoc[]
```

Comment:
seccomp configuration is unrelated to communication with the stack

I think we should just remove the seccomp section altogether. The only time we fork/exec is for the java attacher, and disallowing those syscalls will break that feature. Otherwise if you want to disallow syscalls it's better to apply seccomp rules in the calling environment, e.g. using systemd sandboxing or Docker seccomp config.

Reply:
@bmorelli25 did you see this comment?

Reply:
Whoops. Thanks for the bump on this! Removed.

Reply:
:+1: 

@simitt please shout if you think we should still document seccomp. You may recall we disabled it by default a while back. I think it's still possible to configure, but as mentioned above I think we should guide users to external configuration for this, if needed at all.